pkgs/tools/package-management/disnix/dydisnix/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool , pkgconfig, glib, libxml2, libxslt, getopt, nixUnstable, disnix }:
+
+stdenv.mkDerivation {
+  name = "dydisnix-0.0";
+
+  src = fetchFromGitHub {
+    owner = "svanderburg";
+    repo = "dydisnix";
+    rev = "671db6acbb782404dc45b8b642887e13660a237b";
+    sha256 = "0cydvhjy9jg7gz36wh8crvbwadlv59n498p7rh00kff397lpkrxw";
+  };
+
+  nativeBuildInputs = [ pkgconfig autoconf automake libtool ];
+  buildInputs = [ glib libxml2 libxslt getopt nixUnstable disnix ];
+  preConfigure = ''
+    ./bootstrap
+  '';
+
+  meta = {
+    description = "A toolset enabling self-adaptive redeployment on top of Disnix";
```

Comment:
Maybe add a `longDescription` mentioning that this is experimental software and should not be used in production.

Reply:
added.